Scripts/Generator/Room/RoomBehavior/SpawnRoomBehavior.cs

```diff
@@ -0,0 +1,12 @@
+using Godot;
+
+public class SpawnRoomBehavior : Node
```

Comment:
`SpawnRoomBehaviour` is ambiguous because it might means some other things spawning here. Call it `StartingRoomBehaviour` or `PlayerSpawnRoomBehaviour`?

Reply:
`StartingRoomBehavior` ba